tests/run.rs

```diff
@@ -198,7 +198,7 @@ fn failing() -> R<()> {
             |steps:
             |  - cp
         ",
-        Err(&trim_margin(
+        Expect::error_message(&trim_margin(
```

Comment:
Do we want to move the call to `trim_margin` into `Expect::error_message`? Do we ever use `error_message` without it?

Reply:
You're right, we use `trim_margin` in every case where we use `Expect::error_message` I made this change; and now `Expect::error_message` returns `R<Expect>` instead of `Expect`.